docs/guide/writing-tasks.md

```diff
@@ -585,6 +585,25 @@ task:
 
 You'll be able to manually trigger such paused tasks via Cirrus CI Web UI or directly from GitHub Checks page.
 
+## Required PR Labels
+
+Similar to [manual tasks](#manual-tasks) Cirrus CI can pause execution of tasks until a corresponding PR gets labeled.
+This can be particular useful when you'd like to do an initial review before running all unit and integration
+tests on every [supported platform](supported-computing-services.md). Simply use `required_pr_labels` field to specify
+a list of labels a PR requires to have in order to trigger a task. Here is a simple example of `.cirrus.yml` config
+that automatically runs a linting tool but requires `initial-review` label being presented in order to run tests:
+
+```yaml
+lint_task:
+  # ...
+
+test_task:
+  required_pr_labels: initial-review
```

Comment:
Is it possible to have multiple labels, either inclusive or exclusive?

e.g. only run this task if all labels, or only run this task if at least one of the following labels?

Not sure if that's really needed, more curious

Reply:
You can specify multiple labels and they all should be presented on the PR. For example:

```yaml
build_task:
  required_pr_labels:
    - cla-signed
    - initial-review

task:
  name: Deploy Staging
  depends_on: build
  required_pr_labels:
    - cla-signed
    - initial-review
    - ready-for-staging
```

Reply:
Ahh ok - that would be helpful as an example in the doc